orienteer-core/src/main/java/org/orienteer/core/service/impl/DefaultFilterPredicateFactory.java

```diff
@@ -45,12 +51,23 @@
     }
 
     @Override
+    @SuppressWarnings("unchecked")
     public SerializablePredicate<OProperty> getPredicateForListProperties() {
-        return (prop) -> !(Boolean)CustomAttribute.HIDDEN.getValue(prop);
+        OSecurityUser user = OrienteerWebSession.get().getEffectiveUser();
```

Comment:
Is this performance optimized? Lets measure one day.

Reply:
This is just function composition which creates chain predicate